docs/user-guide.md

```diff
@@ -451,3 +452,16 @@ be used with a `ScheduledSparkApplication`. In most cases, a restart policy of `
 the next run usually picks up where the previous run left anyway. For these reasons, it's often the right choice to use
 a restart policy of `Never` as the example above shows. 
  
+## Customizing Spark Operator
+
+To customize spark operator, you need to go through following steps:
```

Comment:
nit: We use `the Spark Operator` consistently in the doc. Also please add `the` before `following` and some places below, e.g., `the FROM tag`.